guides/V1_Add_Reaction.md

```diff
@@ -0,0 +1,58 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'add-a-reaction'
+is: 'future'
+---
+
+# Add a Reaction
+
+## Concepts
+
+The _addReaction_ mutation allows the user to add a reaction to a message.  The mutation accepts a Message ID (targetId) and a reaction as arguments, and makes the request on behalf of the calling user, adding the specified reaction to the message.
```

Comment:
Maybe we can add to the description for both mutations how the result contains an updated count and whether the user has reacted or not (probably worded better than that 😆).